subscription_test.go

```diff
@@ -0,0 +1,196 @@
+package graphql_test
+
+import (
+	"context"
+	"encoding/json"
+	stdErrors "errors"
```

Comment:
I'd personally add alias to the other `"errors"` package and leave the standard lib one as is.

Reply:
done